src/api/db/migrate/20180221175514_change_configuration_obs_url_default.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeConfigurationObsUrlDefault < ActiveRecord::Migration[5.1]
+  def change
+    change_column_default(:configurations, :obs_url, from: NULL, to: 'https://unconfigured.openbuildservice.org')
```

Comment:
Hm, is this really necessary?

Reply:
What do you mean?

Reply:
It's mentioned in the commit message of 7c27abd4988c0f93fd1b1deb3dce1862ff87a9ae